modules/gdpr_consent/includes/gdpr_consent.agreements.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Pages file for the GDPR Consent module.
+ */
+
+/**
+ * Page callback to list a user's consent agreements.
+ * @todo Finish this.
+ */
+function gdpr_consent_collected_agreements($account) {
+  // Or load it with EFQ.
+  $query = new EntityFieldQuery();
+  $query->entityCondition('entity_type', 'gdpr_consent_agreement');
+
+  $result = $query->execute();
```

Comment:
Presumably this should also filter on user? At which point we'll also want to add it as a key...